test/main/accounts/index.test.js

```diff
@@ -695,3 +698,52 @@ describe('#removeRequests', () => {
     expect(Accounts.removeRequest).not.toHaveBeenCalled()
   })
 })
+
+describe('#signerCompatibility', () => {
+  beforeEach(() => {
+    store.navDash = jest.fn()
+  })
+
+  afterEach(() => {
+    store.removeSigner('12')
+  })
+
+  const signerTypes = ['trezor', 'ledger', 'lattice', 'ring', 'seed', 'aragon']
+  
+  signerTypes.forEach((signerType) => {
+    it(`should open the signer menu when a ${signerType} signer is not available`, (done) => {
+      Accounts.addRequest(request)
+      signers.get.mockReturnValue('12')
+      Accounts.accounts[account.id].signer = '12'
+      Accounts.accounts[account.id].lastSignerType = signerType
+      Accounts.accounts[account.id].signerStatus = 'locked'
```

Comment:
don't need this line, I think it's only used when the account state changes and we want to compare the previous state